extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -206,12 +185,14 @@ export class VisibilityModel {
       this.dispose();
       return;
     }
-    if (this.repeatInterval_) {
+    const totalContinuousMax = Math.max(
+        this.spec_.totalTimeMin, this.spec_.continuousTimeMin);
+    if (totalContinuousMax >= MIN_REPEAT_INTERVAL) {
       const now = Date.now();
       const interval = now - this.firstVisibleTime_;
 
       // Unit in milliseconds
-      const timeUntilRepeat = Math.max(0, this.repeatInterval_ - interval);
+      const timeUntilRepeat = Math.max(0, totalContinuousMax - interval);
       this.ready_ = false;
       dev().assert(!this.scheduleRepeatId_, 'Should not repeat twice');
       this.scheduleRepeatId_ = setTimeout(() => {
```

Comment:
We should be able to get rid of this repeat scheduler since we removed the repeatInterval support.
Instead, we can call refresh after eventResolver .

Reply:
Done

Reply:
...but broke a lot of unit tests. Will work a little while longer, but may need to finish up in the morning.